lbry/tests/unit/test_conf.py

```diff
@@ -222,6 +227,14 @@ def test_max_key_fee_from_args(self):
         c = Config.create_from_arguments(args)
         self.assertEqual(c.max_key_fee, None)
 
+        args = parser.parse_args(['--max-key-fee', 'null'])
+        c = Config.create_from_arguments(args)
+        self.assertEqual(c.max_key_fee, None)
+
+        args = parser.parse_args(['--max-key-fee', ''])
+        c = Config.create_from_arguments(args)
+        self.assertEqual(c.max_key_fee, None)
```

Comment:
I think `null` as an argument is enough, supporting a blank is ambiguous and might have unintended edge cases.